doc/user/content/get-started/install-materialize.md

```diff
@@ -29,13 +29,11 @@ suitable for production deployments</redb>.  See also [Limitations](#limitations
 
 ### Run Materialize in a Docker container
 
-1. In a terminal, issue the following command to run a Docker container from the
-   Materialize image. The command downloads the Materialize image, if one has
-   not been already downloaded.
+1. Go to the
+   [Download](https://materialize.com/download?utm_campaign=General&utm_source=documentation)
```

Comment:
I removed the redirect in https://github.com/MaterializeInc/www/pull/1183

Reply:
👍 And I put back the `docker run` statement instead of going to the Download page.  As an FYI -- this page won't go live until after the /downloads page goes live.